WKData/Sources/WKData/Service/Protocol/WKURLSession.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+public protocol WKURLSession {
+    func wkDataTask(with request: URLRequest, completionHandler: @escaping @Sendable (Data?, URLResponse?, Error?) -> Void) -> WKURLSessionDataTask
```

Comment:
My limited understanding of some of the new Swift concurrency changes (actors/actor isolation/et al) makes it unclear to me if the `@Sendable` here and elsewhere is needed or not.